microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -52,7 +52,7 @@ In the following tutorial, you open DevTools on a "Sluggish Animation" demo page
 
```

Comment:
link to source code at repo:
https://github.com/MicrosoftEdge/Demos/tree/main/devtools-performance-get-started